cmd/wego/main.go

```diff
@@ -16,18 +16,54 @@ import (
 	fluxBin "github.com/weaveworks/weave-gitops/pkg/flux"
 )
 
+var VERSION = "0.0.0-dev.0"
+
 var options struct {
 	verbose bool
 }
 
 var rootCmd = &cobra.Command{
-	Use:   "wego",
-	Short: "Weave GitOps",
+	Use:           "wego [subcommand]",
+	Version:       VERSION,
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Short:         "Weave GitOps",
+	Long:          "Command line utility for managing Kubernetes applications via GitOps.",
+	Example:`
+  # Get verbose output for any wego command
+  wego [command] -v, --verbose
+
+  # Get wego app help
+  wego help app
+
+  # Add gitops application from GitHub repository or local git repo
+  wego app add podinfo
```

Comment:
This would only work if there was a `podinfo` subdirectory contained within the current directory; definitely not from a GitHub repository